src/sparseml/transformers/export.py

```diff
@@ -270,21 +271,17 @@ def create_deployment_folder(
     :param training_directory: path to directory where model files, tokenizers,
         and configs are saved. Exported ONNX model is also expected to be there
     :param onnx_file_name: Name for exported ONNX file in the model directory.
-    :param deployment_files: The set of files that are expected to be present in
-        to the deployment folder once this function terminates.
+    :param deployment_files: optional list of deployment file names to override
+        default file names with.
     :return: path to the valid deployment directory
     """
 
-    if onnx_file_name != MODEL_ONNX_NAME:
-        # replace the default onnx name with the custom one
-        deployment_files.remove(MODEL_ONNX_NAME)
-        deployment_files.update(onnx_file_name)
-
-    if training_directory.split("/")[-1] != "training":
```

Comment:
removing this warning - 1) should be using os or pathlib for working with paths since "/" for directories are os specific 2) default transformers flow right now does not have a directory called "training"